website_customer_also_purchased/__openerp__.py

```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
+{
+    "name": "Website Customer Also Purchased",
+    "version": "1.0",
+    "author": "Vauxoo",
+    "category": "Website",
+    "description": """
+Website Customer Also Purchased
+===============================
+    This app shows the products that were bought in the same
+    purchase order by a other customers related to the current
+    product
+    """,
```

Comment:
This module will be very well accepted publically, can you set description in a separate README.md please.